### PR TITLE
Remove marketdata v1 functions

### PR DESCRIPTION
**Pull request by gnvk**

### CODE REVIEW

File: alpaca_trade_api/stream.py [hunk 1]
```diff
@@ -1,8 +1,3 @@
-"""
-Stream V2
```
Comment (ccnlui) [inline alpaca_trade_api/stream.py:2]: Should we keep `Stream V2` here? I'm fine with remove the rest of the comments, but I wonder if it's better to state this is V2 explicitly.

**Comment by ccnlui** [comment]
I think this paragraph is no longer relevant, we don't have this flag in `Stream`

https://github.com/alpacahq/alpaca-trade-api-python/blob/c0384646329535faeeb3fb82dbf2200d4cd6f488/README.md?plain=1#L382-L385

**Comment by ccnlui** [comment]
This I'm not sure about. Is this still relevant?
https://github.com/alpacahq/alpaca-trade-api-python/blob/c0384646329535faeeb3fb82dbf2200d4cd6f488/README.md?plain=1#L409-L412

**Comment by ccnlui** [comment]
Replace `StreamConn` with `Stream`
https://github.com/alpacahq/alpaca-trade-api-python/blob/c0384646329535faeeb3fb82dbf2200d4cd6f488/README.md?plain=1#L416

**Review by ccnlui — APPROVED** [review]
Just a few comments regarding documentation. LGTM otherwise 👍

File: alpaca_trade_api/stream.py (hunk 1 above)
Comment (gnvk) [reply alpaca_trade_api/stream.py:2]: Actually it isn't. Crypto is in v1beta1, so is the news stream.

**Pull request merged**